### ISSUE TITLE
Temporary files are not removed from cache

### ISSUE THREAD

**Issue by dan-cristian** [TurboGit/hubicfuse#89]
After uploading a large amount of files I run out of local disk space and hubicfuse will crash. 
I realised the problem is that hubicfuse does not delete the cached files stored on the temp folder. 

Is there a local cache file delete function already implemented?

**Comment by TurboGit** [comment]
I'm still not sure why the cached file on the temp folders are not removed when uploaded? Is there a specific reason?

**Comment by dan-cristian** [comment]
I have searched in the code base and could not find any line of code that
expicitly removes local files in temp dir. there is a // todo there
actually saying a better implementation is needed.

**Comment by nephaste** [comment]
same on my side on my Qnap, i have to move the temp file in .hubicfuse file on user space to not fill up ramdisk.

and temp files are staying in this directory not removed

**Comment by TurboGit** [comment]
Can you describe what are the steps to reproduce? I cannot find a single hubic temp file on my systems... And where temps files are stored for you when nothing defined in .hubicfuse (default setting)?

**Comment by dan-cristian** [comment]
My Linux box is booting on a 8 GB USB stick, with 200 MB allocated for /tmp (tmpfs in RAM).
Initially I did not set a temp folder in the .hubicfuse and I quickly got 100% usage on /tmp. I have not looked for the actual files to see where they are placed in /tmp - but I will test it this evening.

After that I added a temp folder in .hubicfuse pointing to a temp folder on a larger hard drive. I can see the accessed files being added on that folder with the .cloudfusexxx naming.
